discovery-provider/alembic/versions/5dd6a55bb738_poa_content_nodes.py

```diff
@@ -0,0 +1,39 @@
+"""POA content nodes
+
+Revision ID: 5dd6a55bb738
+Revises: 5aaa6198def8
+Create Date: 2021-01-12 13:13:37.896953
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5dd6a55bb738'
+down_revision = '5aaa6198def8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('l2_content_nodes',
+        sa.Column('blockhash', sa.String(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('is_current', sa.Boolean(), nullable=False),
+        sa.Column('cnode_id', sa.Integer(), nullable=False),
```

Comment:
jw, what is the cnode_id? / maybe add some comments to what these fields are?

Reply:
https://content-node.audius.co/health_check if you look here, every node of any type within the protocol has a dedicated spID